DependencyInjection/CmfSeoExtension.php

```diff
@@ -290,4 +296,29 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
             $container->removeDefinition($this->getAlias().'.sitemap.guesser.alternate_locales');
         }
     }
+
+    /**
+     * Configuration block to configure form.
+     *
+     * @param array            $config
+     * @param ContainerBuilder $container
+     */
+    private function loadFormConfiguration($config, ContainerBuilder $container)
+    {
+        $seoMetadataClass = null;
+        if (isset($config['data_class'])
+            && isset($config['data_class']['seo_metadata'])
+            && null !== $config['data_class']['seo_metadata']
+        ) {
+            $seoMetadataClass = $config['data_class']['seo_metadata'];
+        } else {
+            if ($container->hasParameter($this->getAlias() . '.backend_type_phpcr')) {
```

Comment:
but why do we have that parameter then? Just for the MappingPath inside the Bundle class?